### PR TITLE
The start of some tests

### PR DESCRIPTION
**Pull request by iffy**
This test fails for me, but I think it should pass, right?

### CODE REVIEW

File: tests/test1.nim [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import macros
+import unittest
+import parsetoml
+import typetraits
+
+test "parseString":
+  let table1 = parsetoml.parseString(""""
```
Comment (kaushalmodi) [inline tests/test1.nim]: You have one extra `"` in there.

File: tests/test1.nim [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+[input]
+file_name = "test.txt"
+""")
+  check table1["input"]["file_name"] == "test.txt"
```
Comment (kaushalmodi) [inline tests/test1.nim]:
```suggestion
  check table1["input"]["file_name"].getStr() == "test.txt"
```

File: tests/test1.nim (hunk 1 above)
Comment (kaushalmodi) [inline tests/test1.nim]:
```suggestion
  let table1 = parsetoml.parseString("""
```

**Comment by iffy** [comment]
Ah! the extra double quote was ruining everything :)  Thank you for finding that.

**Comment by kaushalmodi** [comment]
Manually squashed/rebased and merged.

**Pull request closed without merging**